samples/drivers/video/capture_to_lvgl/sample.yaml

```diff
@@ -0,0 +1,10 @@
+sample:
+  name: Video capture to LVGL
+tests:
+  sample.video.capture_to_lvgl:
+    build_only: true
+    tags:
+      - video
+    platform_allow:
+      - mini_stm32h743
+    depends_on: video
```

Comment:
If `video` not declared by board, CI won't build the test.

Reply:
Sorry, I didn't get the problem. If the board does not declare the `video`, I believe that the `VIDEO_SW_GENERATOR` will be used.

Reply:
I think I understood the issue! :) I added the `video` feature to `mini_stm32h743` board.